components/support/webextensions/src/test/java/mozilla/components/support/webextensions/WebExtensionSupportTest.kt

```diff
@@ -201,10 +202,8 @@ class WebExtensionSupportTest {
         val tabHandlerCaptor = argumentCaptor<TabHandler>()
         WebExtensionSupport.initialize(engine, store)
 
-        store.dispatch(EngineAction.LinkEngineSessionAction(tabId, engineSession)).joinBlocking()
```

Comment:
We don't require the link actions anymore, right? :)

Reply:
Is this related to the initial action patch in the store? :)

Reply:
@Amejia481 right, the engine session is now added directly when we set up the initial state in the test.

Reply:
👍